packages/dx-react-grid/demos/featured/data-editing.md

```diff
@@ -1,5 +1,7 @@
 # React Grid Data Editing
 
-This demo shows the Grid's editing and customization capabilities. You can display custom data types using the `DataTypeProvider` plugin. It is also possible to render a custom component instead of the integrated one.
+This demo shows the Grid's editing and customization capabilities in the [Controlled mode](../../docs/guides/controlled-and-uncontrolled-modes.md). Use this mode when you need to share the Grid's state between other application parts. It also allows you to persist the Grid's state and restore it when required.
```

Comment:
This demo shows how to edit and customize the Grid's data in the Controlled mode?
You can use this mode to share, persist, and restore the Grid's state in other parts of the application?